docs/development-guide.md

```diff
@@ -200,6 +200,32 @@ change it is often desirable to first build a consensus in an issue discussion
 and/or create an initial design doc PR. Once the design has been agreed upon
 one or more PRs implementing the plan can be made.
 
+Pull requests get labelled by maintainers of the repository. Labels help
+reviewers with selecting the pull requests of interest based on components:
+
+* component/build: Issues and PRs related to compiling Ceph-CSI
+* component/cephfs: issues related to CephFS
+* component/deployment: helm chart, kubernetes templates and configuration Issues/PRs
+* component/docs: Issues and PRs related to documentation
+* component/journal: This PR has a change in volume journal
+* component/rbd: issues related to rbd
```

Comment:
s/issues/Issues

Reply:
s/rbd/RBD

Reply:
This is just a copy/paste from the descriptions of the issues. Should I correct them in GitHub too?

Reply:
@nixpanic rest looks good to me

Reply:
@nixpanic if you are fixing this, then sticking to  on of `component/blabla: Aaa aaa` or `component/blabla: aaa aaa` style for all the descriptions would be good.

Rest looks good to me.

Reply:
Thanks, updated the doc and descriptions in GitHub.